Project-18-Food-order-app/src/components/Meal/index.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types";
+
+import Button from "../UI/Button";
+import style from "./meal.module.scss";
+import { URL } from "../../services/constants/constants";
+import { useContext } from "react";
+import { CartContext } from "../../services/stores/CartContext";
+import { currencyFormatter } from "../../services/utils/currencyFormat";
+
+function Meal({ meal }) {
+  const { id, name, price, description, image } = meal;
+
+  const { onAddItem } = useContext(CartContext);
+
+  return (
+    <div className={style["meal-item"]}>
+      <div className={style["img-wrapper"]}>
+        <img src={`${URL}${image}`} alt={name} />
```

Comment:
@sknepali The alt text for your image is the name of the meal. While this is not incorrect, it could be more descriptive. Consider changing it to something like `${name} meal image`